afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/AbstractAppStorage.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage;
+
+import com.powsybl.afs.storage.events.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+public abstract class AbstractAppStorage implements AppStorage {
+
+    protected static final String  NODE_CREATED = "NODE_CREATED";
```

Comment:
Maybe there should be a general topic, or broader topic class.
Here there is no gain to listen to individual event type.

Reply:
to review ..